src/EnergyPlus/DataDaylighting.hh

```diff
@@ -95,88 +96,89 @@ namespace DataDaylighting {
         DElightDaylighting
     };
 
-    // Parameters for "Lighting Control Type"
-    enum class iLtgCtrlType
+    // Parameters for "Lighting Control Type" - these are the values expected by DElight
+    enum class LtgCtrlType
     {
+        Invalid = 0,
```

Comment:
Does this particular one rely on specific values? If not, then `Invalid` should be -1 and the valid enumerations should start at 0.

Reply:
Yes, `LtgCtrlType` enum numeric values are used in the DElight functions (which was discovered when automatic enum values caused diffs). I'm sure that could be altered, but didn't want to go there this round, so add another item to the do-list.